splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -76,6 +76,8 @@ public SpliceCatalogUpgradeScripts(SpliceDataDictionary sdd, TransactionControll
         this.tc=tc;
 
         scripts = new ArrayList<>();
+        addUpgradeScript(baseVersion4, 1996, new UpgradeConglomerateTable(sdd, tc));
```

Comment:
Update 1996 -> 1997

Reply:
making this 1997 would make it run again on 1996 clusters, that already have executed that upgrade script (and if they already executed it, everything is fine). we are not *adding* an upgrade script, just moving it.

Reply:
OK, thanks for the explanation.  If an upgrade fails for some other reason, and we already copied rows from SPLICE_CONGLOMERATE into SPLICE_CONGLOMERATE_SI, would the upgrade try to copy the rows again and then fail the upgrade due to duplicate rows?

Reply:
It looks like UpgradeConglomerateTable has a check to see if SPLICE_CONGLOMERATE_SI is already created, so I guess that's not a problem.